docs/examples/platform/posix/posix_platform_error.hpp

```diff
@@ -27,7 +27,7 @@ class PosixPlatformError final : public libcyphal::transport::IPlatformError
     {
         CETL_DEBUG_ASSERT(err > 0, "");
     }
-    virtual ~PosixPlatformError() noexcept                       = default;
+    virtual ~PosixPlatformError()                                = default;
```

Comment:
I see that you've changed quite a few dtors in this PR to be non-noexcept, but I don't see the point because throwing from a dtor is UB.

Reply:
AFAIK it's redundant to specify `noexcept` at destructors - they all noexcept by default. Please correct me if I'm wrong. Thanks!

Reply:
ah yes you're right

Reply:
btw, here is interesting commit from the Bjarne : https://github.com/isocpp/CppCoreGuidelines/commit/d3c3e3aa986fc9cf6238df2a30a3e071a2f55edd#comments